src/DB/Table/MerchantIssueTable.php

```diff
@@ -61,6 +61,17 @@ public function delete_stale( DateTime $created_before ) {
 		$this->wpdb->query( $this->wpdb->prepare( $query, $created_before->format( 'Y-m-d H:i:s' ) ) ); // phpcs:ignore WordPress.DB.PreparedSQL
 	}
 
+	/**
+	 * Delete product issues for specific products and source.
+	 *
+	 * @param array  $products_ids Array of product IDs to delete issues for.
+	 * @param string $source       The source of the issues. Default is 'mc'.
+	 */
+	public function delete_specific_product_issues( array $products_ids, string $source = 'mc' ) {
```

Comment:
💅  We can add `:void `as a return type in the signature to be consistent with the other functions. `delete_stale`also missed it, but was not introduced in this PR tho